cypress/integration/train/1.2.MyNameIs.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const models = require('../../support/Models')
+const modelPage = require('../../support/components/ModelPage')
+const memoryTableComponent = require('../../support/components/MemoryTableComponent')
+const scorerModal = require('../../support/components/ScorerModal')
+const trainDialogPage = require('../../support/components/TrainDialogsPage')
+const editDialogModal = require('../../support/components/EditDialogModal')
+
+describe("My name is", () =>
+{
+  after(() => { cy.VerifyMonitorFinds() })
+  
+  it('should be able to train', () => 
+  {
+    var modelName = models.ImportModel('Model1-mni', 'Model1-wyn.cl')
+    modelPage.NavigateToTrainDialogs()
+    modelPage.WaitForTrainingStatusCompleted()
+
+    trainDialogPage.CreateNewTrainDialog()
+
+    editDialogModal.TypeYourMessage('My name is David.') // TODO: Add edge cases; 'david', with & without 'period'
+    editDialogModal.VerifyDetectedEntity('name', 'David')
+    editDialogModal.ClickScoreActionsButton()
+    memoryTableComponent.VerifyEntityInMemory('name', 'David')
+    scorerModal.VerifyContainsDisabledAction("What's your name?")
+    scorerModal.ClickAction('Hello David')
+
+    // Wait for the training to complete.
+    // At the time this was added, there is no UI elements to let us know it is complete.
+    cy.wait(20000)
```

Comment:
Is this still true?  I thought you added some code that check training status?

Reply:
Yes, it is still true. 

I did add code to check the training status of the main models page, but this still needs to be done for the the train modal. Also I think you will need to deal with the polling warning if this polling can time out before it is completed.